homework-typescript-intro.ts

```diff
@@ -0,0 +1,92 @@
+// Exercise 1: Check if a number is even or odd
+// Step 1: Declare a variable `num` and assign any number to it.
+// Step 2: Declare a variable `isEven` and use a logical expression to check if `num` is even.
+// Step 3: Log the result
+
+let num : number = 15;
+let isEven: boolean ;
+isEven = (num % 2 ) === 0;
```

Comment:
You don't need to reassign a value. Just add the comparison on variable declaration (upper row)